Tomek Fryczke Fibonacci

```diff
@@ -0,0 +1,9 @@
+import math
+def fib(x):
+  s = math.sqrt(5)
+  f = (1/s) * ( pow(((1+s)/2),x) - pow(((1-s)/2),x) )
+  print(int(f))
+  x = x+1
```

Comment:
ale co sie dzieje tutaj to ja nie wiem:D Nie mozna bylo wrzucic jakiegos komentarza dla glupich programistów? Serio - nawet jesli to implementacja matematyczna która ma rece i nogi, doceniłbym bardzo komentarz mówiący 'o co cho'